core/lib/constants/src/fees/mod.rs

```diff
@@ -46,3 +46,6 @@ pub const GUARANTEED_PUBDATA_IN_TX: u32 = 100000;
 /// of the commitment. The other "36" bytes are mostly an approximation of the amount of gas it takes
 /// to properly hash it and compare with the corresponding L2->L1 message.
 pub const PUBLISH_BYTECODE_OVERHEAD: u32 = 100;
+
+pub const GATEWAY_CALLDATA_PROCESSING_ROLLUP_OVERHEAD: u32 = 200;
+pub const L1_CALLDATA_PROCESSING_ROLLUP_OVERHEAD: u32 = 100;
```

Comment:
`L1_CALLDATA_PROCESSING_ROLLUP_OVERHEAD = 100` looks like a big number for that, can you please share how you derived it?